hdf/src/hchunks.c

```diff
@@ -1270,11 +1249,10 @@ HMCIstaccess(accrec_t *access_rec, /* IN: access record to fill in */
 #if 0 /* dynamic allocation causes a problem on HPUX, removed for now -GV */
```

Comment:
Do we want to keep "#if 0" blocks? OR should this be a "#ifndef HPUX" block

Reply:
At first, I was going to say just remove the "#if 0" blocks completely because HMCIstaccess is using
uint8 c_sp_header[256] = "";
but then I saw c_sp_header being dynamically allocated in HMCcreate also many years ago and also by -GeorgeV, the same author of the above "#if 0" blocks!  So, now I'm going to say switch c_sp_header back to dynamically allocated, i.e., removing the "#if 0"s and the workaround: uint8 c_sp_header[256] = "";

Reply:
I think we should go with new standards and use ifdef machine for special cases for machines we still support.

Reply:
Most of the machine-specific ifdefs are gone now. I left this one alone because I planned to re-enable it. Most "it segfaults on one machine" issues are real memory issues and I suspect we're in better shape now.